pkg/prom/instance/instance.go

```diff
@@ -43,21 +42,6 @@ var (
 
 // Default configuration values
 var (
-	// DefaultRelabelConfigs defines a list of relabel_configs that will
```

Comment:
This actually wasn't doing anything (since it applied post-relabeling) and was lazily copied from Promtail.